pcsx2/CDVD/CDVDdiscReader.cpp

```diff
@@ -186,7 +186,7 @@ s32 CALLBACK DISCopen(const char* pTitle)
 #if defined(_WIN32)
 	std::wstring drive = g_Conf->Folders.RunDisc.GetPath().ToStdWstring();
 #else
-	std::string drive = g_Conf->Folders.RunDisc.GetPath().ToStdString();
+	std::string drive = g_Conf->Folders.RunDisc.GetFullPath().ToStdString();
```

Comment:
Maybe safer to separate macos from linux?

Reply:
`GetPath` only returns the directory part without the filename, I suspect it was not actually working on Linux if there were multiple drives as `/dev/sr1` will just become `/dev` and PCSX2 will fallback to the first drive.